model/countries/AU/AU-descriptions.yaml

```diff
@@ -0,0 +1,3 @@
+short-descriptions:
+  locality1: City - typically reffered as Suburb in AU
+  building: House/Property Number
```

Comment:
Let's drop "/Property" --> `House Number` because it was never used by our surveyed websites.